Technical-Steering-Committee/release.md

```diff
@@ -0,0 +1,79 @@
+
+## Release Approval Procedure
+
+
+### Process when potential stop-ship was found
+
+The TSC via zowe-tsc channel is informed that the potential stop-ship bug was found. The TSC then based on the recommendation from the affected squad/s vote on whether the found bug is a stop ship. The available options are: wait for the fix, return to the previous version of the component, agree it’s not a stop ship. The TSC votes in the channel and the process follows based on the found decision. 
+
+We expect the squads to be able to revert back to the previous version in case it isn’t possible to fix the issue in a time frame to be able to produce release on time. 
+
+
+### Approval of release passing the tests
+
+The TSC is informed by the Release manager that a new release is ready for public distribution. The appointed member of TSC will review the release and ensure that all requirements that are needed for legal and technical release are met. These include, but are not limited to, updated Documentation, correct LICENSE files and code scans showing no known vulnerabilities.
+
+The vote is going to happen in the zowe-tsc channel as explained in the TSC Structure and TSC Charter. The option is to vote for releasing or for stopping the release. When the votes approves the release a TSC member, or the chair, will notify the release manager. The Release manager will then make the artifacts available and notify the OMP Director of the new release to track the LLC release of a new set of binaries.
+
+
+## Release Numbering
+
+Each release will be identified by a version number. These numbers are used according to a specific scheme that will give you additional information about the release.  The version numbers are of the form x.y.z-[GA | beta | yyyymmdd] (Semantic Versioning) or major.minor.micro. The final designation indicates whether this is the official Generally Available (GA) version, a beta version or an interim build. Only releases that end with **-GA** are considered official releases.
+
+* Following semantic versioning the following sequences can be assumed MAJOR.MINOR.PATCH
+  * MAJOR version when you make incompatible API changes
+  * MINOR version when you add functionality in a backwards-compatible manner, and 
+  * PATCH version when you make backwards-compatible bug fixes.
+
+* **GA** indicates that this release is an official, supported version of the Zowe project and is suitable for regular use.
+
+* **beta** means that this build is a candidate build that is complete but is under active development and should not be considered for regular use.
+
+* **yyyymmdd** indicates this is an interim build suitable for experimentation or development. It is not intended for general usage and may contain defects that are known and being worked or unknown.
+The code can be found[here](https://zowe.org/download/).
+
+
+## Release Content
+
+There are two significant and different release artifacts for Zowe. The source release and the "convenience build."
```

Comment:
We currently release these type of bundles which allows download from zowe.org:

- z/OS Component
  * Convenience Build
  * SMP/E Build
- CLI component (Also downloadable from npmjs.org)
  * CLI Core
  * CLI Plugins
- Client SDKs
  * Node.js Client SDK
  * Python Client SDK
- Zowe Explorer (Only downloadable from Visual Studio Code Marketplace)
- Docker build (Tech preview) (Also downloadable from hub.docker.com)
  * amd64
  * x390x

Reply:
I will update the doc